dirsrvtests/tests/suites/setup_ds/db_home_test.py

```diff
@@ -0,0 +1,202 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+
+import pytest
+
+from lib389.utils import *
+from lib389.topologies import topology_st as topo
+from lib389._constants import *
+from lib389.dseldif import *
+from lib389.cli_conf.backend import *
+from lib389.config import BDB_LDBMConfig
+from .... conftest import get_rpm_version
+from lib389.paths import DEFAULTS_PATH
+
+
+pytestmark = [pytest.mark.tier1,
+              pytest.mark.skipif(get_rpm_version("selinux-policy") <= "3.14.3-79" or
+                                 get_rpm_version("selinux-policy") <= "34.1.19-1",
+                                 reason="Will fail because of incorrect selinux labels"),
+              pytest.mark.skipif(ds_is_older('1.4.3.28'), reason='Not implemented')]
+
+DEBUGGING = os.getenv("DEBUGGING", default=False)
+if DEBUGGING:
+    logging.getLogger(__name__).setLevel(logging.DEBUG)
+else:
+    logging.getLogger(__name__).setLevel(logging.INFO)
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.ds2790
+@pytest.mark.bz1780842
+def test_check_db_home_dir_in_config(topo):
+    """Test to check nsslapd-db-home-directory is set to /dev/shm/slapd-instance in cn=config
+
+    :id: 9a1d0fcf-ca31-4f60-8b31-4de495b0b3ce
+    :customerscenario: True
+    :setup: Standalone Instance
+    :steps:
+        1. Create instance
+        2. Check nsslapd-db-home-directory is set to /dev/shm/slapd-instance in cn=config
+    :expectedresults:
+        1. Success
+        2. Success
+    """
+
+    standalone = topo.standalone
+    dbhome_value = '/dev/shm/slapd-{}'.format(standalone.serverid)
```

Comment:
In a container it won't be in /dev/shm, it falls back to /var/lib/...
So perhaps this test module should be skipped when it runs in a container?

Reply:
Thanks, I added another skipif for container.
I build latest package and retested (all passing).
Please review.

Reply:
It still fails in PR CI. The test is not skipped, because systemd is present, even though it is running in a container. See https://github.com/389ds/389-ds-base/blob/af20dce9eba1059eb42c7c45d4e36a071e358f0b/src/lib389/lib389/instance/setup.py#L736-L743
on how to detect if the test is running in a container.